### PR TITLE
[zh] Localize feature-gates/s*.md part 2

### PR DESCRIPTION
**Pull request by 1000Delta**
Related to issue #44410 

Localize part 2 14 files of feature-gates/s*.md

1. service-account-issuer-discovery.md
1. service-account-token-jti.md
1. service-account-token-node-binding-validation.md
1. service-account-token-node-binding.md
1. service-account-token-pod-node-info.md
1. service-app-protocol.md
1. service-internal-traffic-policy.md
1. service-ip-static-subrange.md
1. service-lb-node-port-control.md
1. service-load-balancer-class.md
1. service-load-balancer-finalizer.md
1. service-node-exclusion.md
1. service-nodeport-static-subrange.md
1. service-topology.md

### CODE REVIEW

File: content/zh-cn/docs/reference/command-line-tools-reference/feature-gates/service-account-issuer-discovery.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+---
+<!--
+Enable OIDC discovery endpoints (issuer and JWKS URLs) for the
+service account issuer in the API server. See
+[Configure Service Accounts for Pods](/docs/tasks/configure-pod-container/configure-service-account/#service-account-issuer-discovery)
+for more details.
+-->
+在 API 服务器中为服务账号颁发者启用 OIDC 发现端点
+（颁发者和 JWKS URL）。详情参见
+[为 Pod 配置服务账号](/zh-cn/docs/tasks/configure-pod-container/configure-service-account/#service-account-issuer-discovery)。
```
Comment (windsonsea) [inline content/zh-cn/docs/reference/command-line-tools-reference/feature-gates/service-account-issuer-discovery.md:34]:
```suggestion
在 API 服务器中为服务账号颁发者启用 OIDC 发现端点（颁发者和 JWKS URL）。
详情参见[为 Pod 配置服务账号](/zh-cn/docs/tasks/configure-pod-container/configure-service-account/#service-account-issuer-discovery)。
```

File: content/zh-cn/docs/reference/command-line-tools-reference/feature-gates/service-account-token-jti.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+  - stage: alpha 
+    defaultValue: false
+    fromVersion: "1.29"
+---
+<!--
+Controls whether JTIs (UUIDs) are embedded into generated service account tokens,
+and whether these JTIs are recorded into the Kubernetes audit log for future requests made by these tokens.
+-->
+控制是否将 JTI（UUID）嵌入到生成的服务账号令牌中，
+以及对于这些令牌未来的请求，是否将这些 JTI 记录到 Kubernetes 审计日志中。
```
Comment (windsonsea) [inline content/zh-cn/docs/reference/command-line-tools-reference/feature-gates/service-account-token-jti.md:18]:
```suggestion
以及是否将这些 JTI 记录到 Kubernetes 审计日志中用于这些令牌未来的请求。
```

File: content/zh-cn/docs/reference/command-line-tools-reference/feature-gates/service-internal-traffic-policy.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+    defaultValue: true
+    fromVersion: "1.26"
+    toVersion: "1.27"    
+
+removed: true
+---
+<!--
+Enables the `internalTrafficPolicy` field on Services
+-->
+为服务启用 `internalTrafficPolicy` 字段。
```
Comment (windsonsea) [inline content/zh-cn/docs/reference/command-line-tools-reference/feature-gates/service-internal-traffic-policy.md:27]:
```suggestion
为 Service 启用 `internalTrafficPolicy` 字段。
```

File: content/zh-cn/docs/reference/command-line-tools-reference/feature-gates/service-lb-node-port-control.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+    defaultValue: true
+    fromVersion: "1.24"
+    toVersion: "1.25"    
+
+removed: true
+---
+<!--
+Enables the `allocateLoadBalancerNodePorts` field on Services.
+-->
+为服务启用 `allocateLoadBalancerNodePorts` 字段。
```
Comment (windsonsea) [inline content/zh-cn/docs/reference/command-line-tools-reference/feature-gates/service-lb-node-port-control.md:29]:
```suggestion
为 Service 启用 `allocateLoadBalancerNodePorts` 字段。
```

File: content/zh-cn/docs/reference/command-line-tools-reference/feature-gates/service-load-balancer-class.md [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+    toVersion: "1.25"    
+
+removed: true
+---
+<!--
+Enables the `loadBalancerClass` field on Services. See
+[Specifying class of load balancer implementation](/docs/concepts/services-networking/service/#load-balancer-class)
+for more details.
+-->
+为服务启用 `loadBalancerClass` 字段。
```
Comment (windsonsea) [inline content/zh-cn/docs/reference/command-line-tools-reference/feature-gates/service-load-balancer-class.md:31]:
```suggestion
为 Service 启用 `loadBalancerClass` 字段。
```

File: content/zh-cn/docs/reference/command-line-tools-reference/feature-gates/service-load-balancer-finalizer.md [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+    defaultValue: true
+    fromVersion: "1.17"
+    toVersion: "1.20"    
+
+removed: true
+---
+<!--
+Enable finalizer protection for Service load balancers.
+-->
+为服务负载均衡启用终结器（finalizers）保护。
```
Comment (windsonsea) [inline content/zh-cn/docs/reference/command-line-tools-reference/feature-gates/service-load-balancer-finalizer.md:29]:
```suggestion
为 Service 负载均衡器启用终结器（finalizers）保护。
```

File: content/zh-cn/docs/reference/command-line-tools-reference/feature-gates/service-node-exclusion.md [hunk 7]
```diff
@@ -0,0 +1,32 @@
...
+    fromVersion: "1.21"
+    toVersion: "1.22"    
+
+removed: true
+---
+<!--
+Enable the exclusion of nodes from load balancers created by a cloud provider.
+A node is eligible for exclusion if labelled with "`node.kubernetes.io/exclude-from-external-load-balancers`".
+-->
+启用从云提供商创建的负载均衡中排除节点。
```
Comment (windsonsea) [inline content/zh-cn/docs/reference/command-line-tools-reference/feature-gates/service-node-exclusion.md:30]:
```suggestion
允许从云提供商创建的负载均衡器中排除节点。
```

File: content/zh-cn/docs/reference/command-line-tools-reference/feature-gates/service-topology.md [hunk 8]
```diff
@@ -0,0 +1,25 @@
...
+    defaultValue: false
+    fromVersion: "1.20"
+    toVersion: "1.22"
+
+removed: true
+---
+<!--
+Enable service to route traffic based upon the Node topology of the cluster.
+-->
+允许 Service 基于集群的节点拓扑进行流量路由。
```
Comment (windsonsea) [inline content/zh-cn/docs/reference/command-line-tools-reference/feature-gates/service-topology.md:25]:
```suggestion
允许服务基于集群的 Node 拓扑进行流量路由。
```

File: content/zh-cn/docs/reference/command-line-tools-reference/feature-gates/service-account-token-jti.md (hunk 2 above)
Comment (tengqm) [reply content/zh-cn/docs/reference/command-line-tools-reference/feature-gates/service-account-token-jti.md:18]: @windsonsea 现在的翻译是对的。日志中的数据不会被用来作别的用途。

File: content/zh-cn/docs/reference/command-line-tools-reference/feature-gates/service-account-token-node-binding-validation.md [hunk 9]
```diff
@@ -0,0 +1,16 @@
...
+
+stages:
+  - stage: alpha 
+    defaultValue: false
+    fromVersion: "1.29"
+---
+<!--
+Controls whether the apiserver will validate a Node reference in service account tokens.
+-->
+控制 apiserver 是否会验证服务账号令牌中的 Node 引用。
```
Comment (tengqm) [inline content/zh-cn/docs/reference/command-line-tools-reference/feature-gates/service-account-token-node-binding-validation.md:16]:
```suggestion
控制 API 服务器是否会验证服务账号令牌中的 Node 引用。
```

File: content/zh-cn/docs/reference/command-line-tools-reference/feature-gates/service-account-token-node-binding.md [hunk 10]
```diff
@@ -0,0 +1,16 @@
...
+
+stages:
+  - stage: alpha 
+    defaultValue: false
+    fromVersion: "1.29"
+---
+<!--
+Controls whether the apiserver allows binding service account tokens to Node objects.
+-->
+控制 apiserver 是否允许将服务账号令牌绑定到 Node 对象。
```
Comment (tengqm) [inline content/zh-cn/docs/reference/command-line-tools-reference/feature-gates/service-account-token-node-binding.md:16]:
```suggestion
控制 API 服务器是否允许将服务账号令牌绑定到 Node 对象。
```

File: content/zh-cn/docs/reference/command-line-tools-reference/feature-gates/service-account-token-pod-node-info.md [hunk 11]
```diff
@@ -0,0 +1,18 @@
...
+stages:
+  - stage: alpha 
+    defaultValue: false
+    fromVersion: "1.29"
+---
+<!--
+Controls whether the apiserver embeds the node name and uid
+for the associated node when issuing service account tokens bound to Pod objects.
+-->
+控制 apiserver 在颁发绑定到 Pod 对象的服务账号令牌时，
```
Comment (tengqm) [inline content/zh-cn/docs/reference/command-line-tools-reference/feature-gates/service-account-token-pod-node-info.md:17]:
```suggestion
控制 API 服务器在颁发绑定到 Pod 对象的服务账号令牌时，
```

File: content/zh-cn/docs/reference/command-line-tools-reference/feature-gates/service-topology.md (hunk 8 above)
Comment (tengqm) [reply content/zh-cn/docs/reference/command-line-tools-reference/feature-gates/service-topology.md:25]: 这里用 Service 不翻译是对的。

**Review by windsonsea — COMMENTED** [review]
/lgtm

**Review by ydFu — APPROVED** [review]
/approve

**Pull request merged**